src/assets/scripts/client/commandParser/commandMap.js

```diff
@@ -108,7 +108,6 @@ export const COMMAND_MAP = {
     direct: 'direct',
     pd: 'direct',
     dct: 'direct',
-    abort: 'abort',
```

Comment:
This needs to be present if the command is to register upon a user entering it

Reply:
Hurray, I'm an idiot

Reply:
@sam-irl not an idiot at all.  This is why we do pull requests.  Another set (or sets) of eyes is always a good idea!